<launch>
  <!-- Map server -->
  <arg name="map_file" default="/home/team6/maps/playground.yaml" />
  <node name="map_server" pkg="map_server" type="map_server" args="$(arg map_file)" />

  <!-- Our own servers-->
  <node pkg="recycle_ui" type="map_annotator_server.py" name="recycle_map_annotator" args="$(env HOME)/our_pose_db_name.db"/>
  <node pkg="recycle_ui" type="classification_server.py" name="classification_server" />
  <node pkg="recycle_ui" type="add_item_server.py" name="add_item_server" />

  <!-- Web video -->
  <node pkg="web_video_server" type="web_video_server" name="web_video_server">
    <param name="port" value="8000" />
  </node>

  <!-- Web prereqs -->
  <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
  <node pkg="tf2_web_republisher" type="tf2_web_republisher" name="tf2_web_republisher" />
  <node pkg="interactive_marker_proxy" type="proxy" name="interactive_marker_proxy_basic_controls">
    <remap from="target_frame" to="/map" />
    <remap from="topic_ns" to="/map_annotator/map_poses" />
    <param name="update_rate" value="10.0" />
  </node>
  <!-- <node pkg="collada_web_server" type="run_caddy.bash" name="collada_caddy" cwd="node" /> -->
</launch>
